src/Microsoft.ML.Data/Data/Conversion.cs

```diff
@@ -1673,5 +1674,9 @@ public void Convert(in TX src, ref SB dst)
         public void Convert(in BL src, ref R8 dst) => dst = System.Convert.ToDouble(src);
         public void Convert(in BL src, ref BL dst) => dst = src;
         #endregion FromBL
+
+        #region ToDT
+        public void Convert(in DT src, ref DT dst) => dst = src;
```

Comment:
>public void Convert(in DT src, ref DT dst) => dst = src; [](start = 8, length = 56)

Does not have test coverage as per [https://codecov.io/gh/dotnet/machinelearning/pull/4273/diff](https://codecov.io/gh/dotnet/machinelearning/pull/4273/diff)

Reply:
Added a unit test for the new DateTime to DateTime conversion method.

---
In reply to: [330341048](https://github.com/dotnet/machinelearning/pull/4273#discussion_r330341048) [](ancestors = 330341048)

Reply:
Thanks, PH!

---
In reply to: [330650008](https://github.com/dotnet/machinelearning/pull/4273#discussion_r330650008) [](ancestors = 330650008,330341048)